Index: TandooriRecipe/Controllers/RecipeController.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>using Microsoft.AspNetCore.Mvc;\n\nnamespace TandooriRecipe.Controllers\n{\n    public class RecipeController : Controller\n    {\n        // GET\n        public IActionResult Index()\n        {\n            return View();\n        }\n    }\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- TandooriRecipe/Controllers/RecipeController.cs	(date 1573092497000)
+++ TandooriRecipe/Controllers/RecipeController.cs	(date 1573092399966)
@@ -5,9 +5,9 @@
     public class RecipeController : Controller
     {
         // GET
-        public IActionResult Index()
-        {
-            return View();
-        }
+//        public IActionResult Index()
+//        {
+//            return View();
+//        }
     }
 }
\ No newline at end of file
